sklearn/metrics/classification.py

```diff
@@ -2389,8 +2370,28 @@ def brier_score_loss(y_true, y_prob, sample_weight=None, pos_label=None):
     assert_all_finite(y_prob)
     check_consistent_length(y_true, y_prob, sample_weight)
 
+    labels = np.unique(y_true)
+    if len(labels) > 2:
+        raise ValueError("Only binary classification is supported. "
+                         "Provided labels %s." % labels)
+    if y_prob.max() > 1:
+        raise ValueError("y_prob contains values greater than 1.")
+    if y_prob.min() < 0:
+        raise ValueError("y_prob contains values less than 0.")
+
     if pos_label is None:
-        pos_label = y_true.max()
+        if (np.array_equal(labels, [0, 1]) or
```

Comment:
Share this code with `_binary_clf_curve`? And document the behaviour?

Reply:
This seems difficult since the definition of pos_label=None is different?